Project/Galagram/ViewModel/Commands/User/Setting/LoadNewAvatarCommand.cs

```diff
@@ -55,9 +55,9 @@ public override void Execute(object parameter)
                 // create temp folder, and log it
                 settingViewModel.CreateTempFolderIfNotExist();
 
-                // get random name
+                // get avatar path
                 string localAvatarPath = fileNames[0];
-                string serverTempAvatarPath = settingViewModel.GetRandomTempFileName(fileExtension: System.IO.Path.GetExtension(localAvatarPath));
```

Comment:
I am not sure, that you doing this right. Why you remove random name generator? What happens if user upload photo with the same name?